lib/championer_one_web/templates/layout/app.html.eex

```diff
@@ -13,10 +13,15 @@
   </head>
 
   <body class="yellow lighten-4">
-   
-
-    <%= render "background_video.html", assigns %>
-    <div class="container">
+    <header class="header">
+      <%= render "navbar.html", assigns %>
+    </header>     
+    <%= if @view_template == "index.html" do %>
```

Comment:
Instead of the nested if statement, what if you try this....

```elixir 
<%= if @view_template == "index.html" && @view_module == ChampionerOneWeb.PageView  do %>
  <%= render "background_video.html", assigns %>
<% end %>